File: Nim/Nim_101/USD_to_EUR.py
#
eur_to_usd = 1.08
usd_to_eur = 0.92
#
mult = 0.00
result = 0.00
#
Input_coin = ""
Output_coin = ""
#
while True:
    choice = int(input("Welcome\n1. EUR to USD\n2. USD to EUR\n--->"))
    #
    if choice == 1:
        Input_coin = "EUR"
        Output_coin = "USD"
        mult = eur_to_usd
        break
    #
    elif choice == 2:
        Input_coin = "USD"
        Output_coin = "EUR"
        mult = usd_to_eur
        break
    #
    else:
        print("Syntax ERR\nPlease try again\n")
#
amount = float(input("\nInsert amount in {}\n--->".format(Input_coin)))
#
result = amount * mult
print("\n---> {} {}".format(result, Output_coin))
#

File: Nim/Nim_101/USD_to_EUR.nim
#
import strutils;
#
var eur_to_usd = 1.08;
var usd_to_eur = 0.92;
#
var mult = 0.00;
var result = 0.00;
#
var Input_coin: string = "";
var Output_coin: string = "";
#
while true:
    echo ("Welcome\n1. EUR to USD\n2. USD to EUR");
    var choiceStr = readLine(stdin);
    var choice = choiceStr.parseInt();
    #
    if choice == 1:
        Input_coin = "EUR";
        Output_coin = "USD";
        mult = eur_to_usd;
        break;
    #
    elif choice == 2:
        Input_coin = "USD";
        Output_coin = "EUR";
        mult = usd_to_eur;
        break;
    #
    else:
        echo "Syntax ERR\nPlease try again\n";
#
echo "\nInsert amount in ", Input_coin, ":";
var amountStr = readLine(stdin);
var amountInt = amountStr.parseInt();
var amount = amountInt.float;
#
result = amount * mult;
echo ("\n", result, " ", Output_coin);
#
